lib/Utils/Asset.php

```diff
@@ -17,21 +17,59 @@ class Asset
     protected static $assetManifest;
     protected static $loadFromCdn = false;
 
+    /**
+    * Gets the asset's url.
+    *
+    * @since 0.1.0
+    *
+    * @param string $asset The required asset.
+    *
+    * @return string/boolean Returns the url or false if the asset is not found.
+    */
     public static function requireUrl($asset)
     {
         return self::get('url', $asset);
     }
 
+    /**
+    * Gets the asset's path.
```

Comment:
I would add the keyword "absolute" here to be more specific (could be the relative path in theory)